lib/crowdin/adapter.rb

```diff
@@ -137,30 +137,42 @@ def translations_for_file(file_id, language)
       translations_for_files([file_id], language)
     end
 
-    # Given a hash of objects that have been successfully sync'd, and all the +available_locales+,
-    # delete all the source files that have successfully been sync'd in every locale.
+    def cleanup_translations(files_to_cleanup)
+      if files_to_cleanup.present?
+        failed_files = safe_file_iteration(files_to_cleanup) { |file_id| @client.delete_file(file_id) }
+      else
+        failed_files = nil
+      end
+
+      ReturnObject.new(nil, failed_files)
+    end
+
+    # Given a model_name and id, delete all the source files that have been sync'd in every locale.
```

Comment:
there's no notion of locale in this method, right? could we adjust the comment to reflect that it just deletes the source file for the given model_name / id. Also, could we rename to just `delete_source_strings_for_model` / `delete_source_files_for_model`.

Then we can rename `cleanup_translations` => `delete_source_files` / `delete_source_strings`
and rename `cleanup_specific_translations` => `cleanup_translations`

In this world, the idea of "cleaning up" refers to checking the locales and only deleting source files if translations were completely sync'd for all languages.

Hope that makes sense.

Reply:
Yeah no notion of locale in this method, missed taking out that locale part of the description after copying from cleanup_translations. Thanks for the additional context on naming, will make the changes and update specs